docs/pages/development/installation.md

```diff
@@ -58,70 +53,48 @@ To do that, open Xcode and go to General > Deployment Info, and select an iOS ve
 
 <img src="/static/images/client/check_ios_version.png" style={{maxWidth: "100%" }}/>
 
-</Tab >
-
-</Tabs >
-
 ### Android
 
 No additional changes are needed to install the package on Android.
 
-## 2. Basic configuration
+</Tab >
 
-Development builds use deep links to open projects from the QR code. If you have added a custom deep link scheme to your project, your development build will use it. However, if this isn't the case, you need to configure the deep link support for your app. The `uri-scheme` package will do this for you once you have chosen a scheme.
+</Tabs >
 
-<InstallSection packageName="expo-development-client" cmd={["npx uri-scheme add <your scheme>"]} hideBareInstructions />
+## 2. Basic configuration
```

Comment:
```suggestion
## 3. Basic configuration
```

Also, can I please ask you to change the numbered list pattern for the rest of the sections? 🙏 

I'd have done it myself but I can't review the unchanged diff 😄